### PR TITLE
Updated CSS on Docs site

### PR DESCRIPTION
**Pull request by AustinLewis1**
**Asana Ticket:** https://app.asana.com/0/1141690501624281/1161901538841625

**Description:** We are updating the CSS of docs.activestate.com. The site is being upgraded to run on the Hugo theme, and we are going to customize the theme to match the design system being used on the new activestate.com. 

The changes done were mostly typographical where the headings were change to be the Roboto font and the body was changed to the Open Sans font. Links were given a color update to match the blue in the new brand colors. 

**There is a page titled "Style Guide" that I am unsure of what to do with. I will update this PR after meeting to review it.** 

**Screenshots:**
image

image

image

image

**Logo with left-alignment:**

image

### CODE REVIEW

**Comment by flukeout** [comment]
Hey @AustinLewis1 some quick notes. I haven't had a chance to get this running locally with your changes yet. Just looking over the screencaps so far.

* Can we try matching the grey and red in the docs sidebar to the exact color values in the latest mocks? 
  * https://redpen.io/po9635550f1ce48ff4
  * The red especially near the AcitveState logo is noticeably different
* For the main headings like "State Tool CLI" let's not use all caps and left-align please
* Try working in "Docs" or "Documentation" below the ActiveState logo somewhere and probably make the AS logo a little bit smaller.

**Comment by flukeout** [comment]
@AustinLewis1 dropped some feedback into https://docs.google.com/document/d/1jeljE1h5opkSHHHCazIBiB_YX8i4XSD0UvkJlXNqMYU/edit figured that was easier, let me know if you've got any questions or wanna discuss.

File: layouts/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```
Comment (flukeout) [inline layouts/partials/header.html:1]: @AustinLewis1 is all of this content here just so we can change that one icon from the "forking" to the "edit pencil"?

File: docs/css/theme.css [hunk 2]
```diff
@@ -20,11 +20,10 @@
     margin: 2px 0 -2px 0;
 }
 @font-face {
-    font-family: 'Novacento Sans Wide';
-    src: url("../fonts/Novecentosanswide-UltraLight-webfont.eot");
-    src: url("../fonts/Novecentosanswide-UltraLight-webfont.eot?#iefix") format("embedded-opentype"), url("../fonts/Novecentosanswide-UltraLight-webfont.woff2") format("woff2"), url("../fonts/Novecentosanswide-UltraLight-webfont.woff") format("woff"), url("../fonts/Novecentosanswide-UltraLight-webfont.ttf") format("truetype"), url("../fonts/Novecentosanswide-UltraLight-webfont.svg#novecento_sans_wideultralight") format("svg");
+    font-family: 'Roboto';
+    src: url("../fonts/Roboto-Light.ttf");
```
Comment (flukeout) [inline docs/css/theme.css]: Let's include both Open Sans and Roboto via importing from Google Fonts.

**Comment by flukeout** [comment]
There are a bunch of `.DS_Store` files in this PR. Let's not include those. One way to do that is to add a `.gitignore` file to the repo and put `.DS_Store` in it. You can reference dashboard repo to see how to set it up and then commit that.

File: layouts/partials/header.html (hunk 1 above)
Comment (AustinLewis1) [reply layouts/partials/header.html:1]: Yes it is

File: static/css/theme-activestate.css [hunk 3]
```diff
@@ -104,4 +183,4 @@ a:hover {
 
 #sidebar hr {
     border-color: var(--MENU-SECTION-HR-color);
-}
+}
```
Comment (flukeout) [inline static/css/theme-activestate.css]: Can you add a line break after this last rule so the file ends with a blank line? github likes that for some reason.

**Review by flukeout — CHANGES_REQUESTED** [review]

File: docs/css/theme.css [hunk 4]
```diff
@@ -1121,7 +1118,7 @@ pre .copy-to-clipboard:hover {
 }
 
 #sidebar #shortcuts h3 {
-    font-family: "Novacento Sans Wide", "Helvetica", "Tahoma", "Geneva", "Arial", sans-serif;
+    font-family: 'Roboto', sans-serif;
```
Comment (flukeout) [inline docs/css/theme.css]: Can we standardize on Double quotes round the font name as above?

File: static/css/theme-activestate.css [hunk 5]
```diff
@@ -5,20 +5,20 @@
 
 :root{
```
Comment (flukeout) [inline static/css/theme-activestate.css:8]: It looks like we're specifying font-family changes here AND in the theme above. Is that necessary? Can we just specify the headings and fonts here and not modify the theme above when we don't have to?

File: docs/css/theme.css [hunk 6]
```diff
@@ -19,13 +19,11 @@
...
 }
-@font-face {
-    font-family: 'Novacento Sans Wide';
-    src: url("../fonts/Novecentosanswide-UltraLight-webfont.eot");
-    src: url("../fonts/Novecentosanswide-UltraLight-webfont.eot?#iefix") format("embedded-opentype"), url("../fonts/Novecentosanswide-UltraLight-webfont.woff2") format("woff2"), url("../fonts/Novecentosanswide-UltraLight-webfont.woff") format("woff"), url("../fonts/Novecentosanswide-UltraLight-webfont.ttf") format("truetype"), url("../fonts/Novecentosanswide-UltraLight-webfont.svg#novecento_sans_wideultralight") format("svg");
-    font-style: normal;
-    font-weight: 200;
-}
+
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto:wght@300&display=swap');
```
Comment (flukeout) [inline docs/css/theme.css]: Should we put this in `theme-activestate.css`?

File: static/css/theme-activestate.css [hunk 7]
```diff
@@ -32,15 +32,94 @@
 
 body {
     color: var(--MAIN-TEXT-color) !important;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 300;
+    line-height: 1.6;
```
Comment (flukeout) [inline static/css/theme-activestate.css:38]: Do we need units for `line-height`?

File: static/css/theme-activestate.css [hunk 8]
```diff
@@ -32,15 +32,94 @@
...
 }
 
-h2, h3, h4, h5 {
+h1, h2, h3, h4, h5 {
     color: var(--MAIN-TITLES-TEXT-color) !important;
+    font-family: 'Roboto', sans-serif;
+    text-rendering: optimizeLegibility;
+    color: #5e5e5e;
+    font-weight: 300;
+    letter-spacing: 0px;
```
Comment (flukeout) [inline static/css/theme-activestate.css:53]: We can just use `0` here without units.

File: static/css/theme-activestate.css [hunk 9]
```diff
@@ -32,15 +32,94 @@
...
+    font-size: small;
+    font-style: normal !important;
+    margin-top: -24px !important;
+    padding-bottom: 24px !important;
+}
+
+li {
+    padding: 0 0 1.5rem 0;
+}
+
```
Comment (flukeout) [inline static/css/theme-activestate.css:80]: Nuke out double spaces. Lets go with single line breaks between rules.

File: static/css/theme-activestate.css [hunk 10]
```diff
@@ -32,15 +32,94 @@
...
+pre .copy-to-clipboard:hover {
+    background-color: #77A2DB;
+    color: #fff;
+}
+
+.copy-to-clipboard {
+    background-color: #E8ECFE;
+}
```
Comment (flukeout) [inline static/css/theme-activestate.css:119]: Linting: Add line break after this rule.

File: static/css/theme-activestate.css [hunk 11]
```diff
@@ -64,10 +143,10 @@ a:hover {
     content: "";
     height: 1px;
     width: 0%;
-    -webkit-transition: width 0.5s ease;
-    -moz-transition: width 0.5s ease;
+    -webkit-transition: width 0s ease;
+    -moz-transition: width 0s ease;
     -ms-transition: width 0.5s ease;
-    transition: width 0.5s ease;
+    transition: 0
```
Comment (flukeout) [inline static/css/theme-activestate.css]: Can you double-check if transition 0 is valid? Might be something like `transition: none` and also, we should probably match the rule for all of the prefixed ones as well.

File: static/css/theme-activestate.css [hunk 12]
```diff
@@ -64,10 +143,10 @@ a:hover {
     content: "";
     height: 1px;
     width: 0%;
-    -webkit-transition: width 0.5s ease;
-    -moz-transition: width 0.5s ease;
+    -webkit-transition: width 0s ease;
+    -moz-transition: width 0s ease;
     -ms-transition: width 0.5s ease;
```
Comment (flukeout) [inline static/css/theme-activestate.css]: Any reason the `-ms` one still has .5s?

**Review by flukeout — CHANGES_REQUESTED** [review]

**Review by flukeout — APPROVED** [review]

**Pull request merged**